src/transformers/onnx/convert.py

```diff
@@ -350,6 +363,8 @@ def validate_model_outputs(
     session = InferenceSession(onnx_model.as_posix(), options, providers=["CPUExecutionProvider"])
 
     # Compute outputs from the reference model
+    if is_torch_available() and issubclass(type(reference_model), PreTrainedModel):
+        reference_model.to("cpu")
```

Comment:
Should this be `device` here or not?
```suggestion
        reference_model.to(device)
```

Reply:
In `validate_model_outputs`, the generated dummy inputs are on CPU, as in the `export` the `model` has been sent to CUDA if `device="cuda"`, here we need to send it back to CPU so that it can be on the same device as the dmmy inputs.

Reply:
Ah perfect, please ignore my comment then :)